generators/pythonv2/codegen/src/ast/Variable.ts

```diff
@@ -0,0 +1,37 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+import { Annotation } from "./Annotation";
+
+export declare namespace Variable {
+    interface Args {
+        /* The name of the variable */
+        name: string;
+        /* The type annotation of the variable */
+        type?: Annotation;
+        /* The initializer for the variable */
+        initializer?: string;
```

Comment:
[nit] I'm surprised that initializer is a `string` and not an `AstNode`

Reply:
Hm I see a couple different patterns for initializers in csharp.

String: https://github.com/fern-api/fern/blob/main/generators/csharp/codegen/src/ast/Parameter.ts#L14
Codeblock: https://github.com/fern-api/fern/blob/main/generators/csharp/codegen/src/ast/Field.ts#L32

I'd be interested in @dsinghvi's preference here.

Reply:
And introduce a `CodeBlock AST Node` which can either be a string or can be a lambda that takes a writer. basically a glorified function that will allow you to write any code you want

Reply:
When do you guys use `CodeBlock` vs stuff like [this](https://github.com/fern-api/fern/pull/4919/files#diff-377cd78484956f0278d9409f9c983abe8361f18f6f83d95ef7b557fc6a05e9ddR7)

Since this initializer is basically code, I would think we would want it all to be ast traversable